[Run] Updated tooltip styling and fixed casing

niels9001 opened the pull request:
## Summary of the Pull Request
- Alignment in tooltip styles: now following the UWP tooltip styling for all themes. (#4711)

Dark theme
![Tooltip](https://user-images.githubusercontent.com/9866362/86967481-f6d3ef00-c16a-11ea-80f8-6a0f79347a51.gif)

Light theme
![image](https://user-images.githubusercontent.com/9866362/86967606-24209d00-c16b-11ea-93c6-b7f2ec5b6a78.png)

- Fixed casing issue (#4842)

## PR Checklist
* [X] Applies to #4711 #4842
* [X] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA

niels9001 committed:
Added custom tooltip style (with theming support) that represents the UWP tooltip style.

niels9001 committed:
Removed unnecessary space in text

niels9001 committed:
Fixed run as admin casing

dsrivastavv reviewed (approved):
LGTM!

alekhyareddy28 reviewed (changes requested):
Regarding changing the case of `Run As Administrator`, it is also present in the indexer plugin. Can that be modified as well?

htcfreek commented:
> Regarding changing the case of `Run As Administrator`, it is also present in the indexer plugin. Can that be modified as well?

Can we have generally strings for such reused functions used in all plugins instead have one string for every plugin?

niels9001 commented:
> Regarding changing the case of `Run As Administrator`, it is also present in the indexer plugin. Can that be modified as well?

![image](https://user-images.githubusercontent.com/9866362/87152589-43781100-c2b6-11ea-95ac-4bec594b5291.png)

Happy to update it, but I can't seem to find it. I searched on all strings and think I updated them everywhere. Let me know which file I'm still missing and I'll fix it.


> > Regarding changing the case of `Run As Administrator`, it is also present in the indexer plugin. Can that be modified as well?
> 
> Can we have generally strings for such reused functions used in all plugins instead have one string for every plugin?

These are strings listed in separate plug-ins. Let's keep it that way, since these are plug-ins they can be easily added, removed or updated.

htcfreek commented:
@niels9001 
Is the code from #4807 missing in your PR?

niels9001 committed:
Merge branch 'master' into user/niels9001/run-tooltipstyling

niels9001 committed:
Fixed casing for Indexer plugin

niels9001 commented:
> @niels9001
> Is the code from #4807 missing in your PR?

Ah, missed that! Thanks for the headsup!

@alekhyareddy28 I have updated Indexer strings with the correct casing as well.

alekhyareddy28 reviewed (approved):
LGTM. Thanks for updating the case @niels9001.

alekhyareddy28 commented:
> > Regarding changing the case of `Run As Administrator`, it is also present in the indexer plugin. Can that be modified as well?
> 
> Can we have generally strings for such reused functions used in all plugins instead have one string for every plugin?

I'm not sure off the top of my head, but I think we can look into that. It would be nice to remove duplicate strings.
